internal/services/subscription/subscription_resource.go

```diff
@@ -118,6 +118,24 @@ func resourceSubscription() *pluginsdk.Resource {
 				Computed:    true,
 			},
 
+			"subscription_owner_id": {
+				Type:         pluginsdk.TypeString,
+				Description:  "Owner ID of the subscription",
+				Optional:     true,
+				ForceNew:     true,
```

Comment:
This should not be `ForceNew`, the update method also need to be updated below to send the update when the value is changed, changing the ownerId should not remove the subscription afaik.

Reply:
Changing the owner id does not remove the subscription but it is also not working. It seems that the API accepts changing the owner ID but it does not actually update the IAM role assignments. It looks like it silently ignores it. Therefore, I set `ForceNew` to true to make it clear to the user that he has to re-create the subscription when he wants to change the owner. Is there a better way to achieve this?